src/components/charts/SummaryPieChart/index.jsx

```diff
@@ -56,13 +56,18 @@ class SummaryPieChart extends React.Component {
                         {entry.name}
                       </div>
                       <div className='summary-pie-chart__legend-item-value form-special-number'>
-                      <span className='summary-pie-chart__legend-item-value-number'>
-                        { helper.numberWithCommas(entry.value) }
-                      </span> 
-                      <span className='summary-pie-chart__legend-item-value-percentage'> ({
-                          helper
-                            .percentageFormatter(this.props.showPercentage)(entry[dataKey])
-                      })</span>
+                        <span className='summary-pie-chart__legend-item-value-number'>
+                          { helper.numberWithCommas(entry.value) }
+                        </span>
+                        <span className='summary-pie-chart__legend-item-value-percentage'>
+                          {' '}
+(
+                          {
+                            helper
+                              .percentageFormatter(this.props.showPercentage)(entry[dataKey])
+                          }
+)
```

Comment:
what's going on with this linting 😆

Reply:
good catch 🤦 . I think there was some wrong syntax in the code (a redundent `()`) which confuses the linter. Fixed now